pennylane/ops/qubit/special_unitary.py

```diff
@@ -0,0 +1,346 @@
+
+    Args:
+        num_wires (int): The number of wires, or number of letters per word.
+
+    Returns:
+        list[str]: All Pauli words on ``num_wires`` qubits, except from the identity.
+
+    There are :math:`d=4^n-1` Pauli words that are not the identity. They are ordered
+    (choose the description that suits you most)
+
+      - lexicographically.
+
+      - such that the term acting on the last qubit changes fastest, the one acting on the first
+        qubit changes slowest when iterating through the output.
+
+      - such that the basis index, written in base :math:`4`, contains the indices for the list
+        ``["I", "X", "Y", "Z"]``, in the order of the qubits
+
+      - such that for three qubits, the first Pauli words are
+        ``"IIX", ""IIY", "IIZ", "IXI", "IXX", "IXY", "IXZ", "IYI"...``
+
+    **Example**
+
+    >>> pauli_words(1)
+    ['X', 'Y', 'Z']
+    >>> len(pauli_words(3))
+    63
+    """
+    return ["".join(letters) for letters in list(product(_pauli_letters, repeat=num_wires))[1:]]
+
+
+def special_unitary_matrix(theta, num_wires):
+    r"""Compute the matrix of an element in SU(N), given by the Pauli basis coordinated
+    of the associated Lie algebra element.
+
+    Args:
+        theta (tensor_like): Pauli coordinates of the exponent :math:`A(\theta)`.
+        num_wires (int): The number of wires the matrix acts on.
+
+    Returns:
+        tensor_like: matrix of the special unitary corresponding to ``theta``. It
+            has the shape ``(2**num_wires, 2**num_wires)``.
+
+    The :math:`4^n-1` Pauli basis elements of the Lie algebra :math:`\mathfrak{su}(N)`
+    for :math:`n` qubits are
+    :math:`P_m\in\{I, X, Y, Z\}^{\otimes n}\setminus\{I^{\otimes n}\}`, and the special
+    unitary matrix is computed as
+
+    .. math::
+
+        U(\theta) = \exp(i\sum_{m=1}^d \theta_m P_m)
+
+    See :func:`~.ops.qubit.matrix_ops.pauli_words` for the ordering of Pauli words.
+
+    .. note::
+
+        Note that this method internally handles a complex-valued tensor of size
+        ``(4**num_wires, 2**num_wires, 2**num_wires)``, which requires at least
+        ``4 ** (2 * num_wires - 13)`` GB of memory (at default precision).
+    """
+    if num_wires > 5:
+        theta = qml.math.hstack([[0], theta])
+        A = sum(
+            t * reduce(np.kron, letters)
+            for t, letters in zip(theta, product(_pauli_matrices, repeat=num_wires))
+        )
+    else:
+        A = qml.math.tensordot(theta, pauli_basis(num_wires), axes=[[-1], [0]])
+    return qml.math.expm(1j * A)
+
+
+class SpecialUnitary(Operation):
+    r"""Gate from the group :math:`SU(N)` with :math:`N=2^n` for :math:`n` qubits.
+
+    We use the following parametrization of a special unitary operator:
+
+    .. math::
+
+        U(\theta) &= e^{A(\theta)}\\
+        A(\theta) &= \sum_{m=1}^d i \theta_m P_m\\
+        P_m &\in {I, X, Y, Z}^{\otimes n} \setminus \{I^{\otimes n}\}
```

Comment:
```suggestion
        P_m &\in \{I, X, Y, Z\}^{\otimes n} \setminus \{I^{\otimes n}\}
```
not sure this renders either

Reply:
This works :)